spinnaker-monitoring-third-party/third_party/prometheus/front50-microservice-dashboard.json

```diff
@@ -769,15 +769,15 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum($Function(front50:storageServiceSupport:numAdded{instance=~\"$Instance\"}[$SamplePeriod])) by (objectType)",
+              "expr": "sum($Function(front50:storageServiceSupport:numAdded_total{instance=~\"$Instance\"}[$SamplePeriod])) by (objectType)",
               "intervalFactor": 2,
               "legendFormat": "add {{objectType}}",
               "metric": "",
               "refId": "A",
               "step": 40
             },
             {
-              "expr": "-1 * sum($Function(front50:storageServiceSupport:numRemoved{instance=~\"$Instance\"}[$SamplePeriod])) by (objectType)",
+              "expr": "-1 * sum($Function(front50:storageServiceSupport:numRemoved_total{instance=~\"$Instance\"}[$SamplePeriod])) by (objectType)",
```

Comment:
missing an update to `numUpdated` in this file

Reply:
Thanks. Fixed.